### PR TITLE
feat(supplier): add

### PR DESCRIPTION
**Pull request by AzizSandid**
## What does this do?
Finish my part in the supplier sprint.

### CODE REVIEW

File: src/modules/supplier/profile/transaction-history .tsx
Comment (guardian-001) [inline src/modules/supplier/profile/transaction-history .tsx:1]: translate

**Review by guardian-001 — CHANGES_REQUESTED** [review]

File: src/modules/supplier/profile/subscription-management.tsx
[the remark this answers is not in the document]
Comment (AzizSandid) [reply src/modules/supplier/profile/subscription-management.tsx:1]: fixed

File: src/modules/supplier/profile/bank-card-form.tsx
[the remark this answers is not in the document]
Comment (AzizSandid) [reply src/modules/supplier/profile/bank-card-form.tsx:1]: fixed

File: src/modules/supplier/profile/bank-transfer.tsx
[the remark this answers is not in the document]
Comment (AzizSandid) [reply src/modules/supplier/profile/bank-transfer.tsx:1]: fixed

File: src/modules/supplier/profile/transaction-history .tsx
Comment (AzizSandid) [reply src/modules/supplier/profile/transaction-history .tsx:1]: fixed

**Review by guardian-001 — APPROVED** [review]

**Pull request merged**